rpm/iptables.spec

```diff
@@ -47,7 +44,6 @@ network and you are using ipv6.
 
 %package doc
 Summary:   Documentation for %{name}
-Group:     Documentation
```

Comment:
`doc` sub-package can be marked with `BuildArch: noarch`.

Reply:
Yep, added with 4d5cb7d932a61067afa064f44ac0d7d118ba5690